src/shared/locales/en-US/preferences.js

```diff
@@ -6,17 +6,17 @@ export default {
   'discard': 'Discard',
   'startup': 'Startup',
   'open-at-login': 'Open at login',
-  'keep-window-state': 'Restore size and position of the window',
-  'auto-resume-all': 'Auto resume all unfinished tasks',
-  'default-dir': 'Default Dir',
-  'mas-default-dir-tips': 'Due to the sandbox permissions restrictions of the App Store, the default download dir is recommended to be set to Downloads directory',
-  'task-manage': 'Task Manage',
+  'keep-window-state': 'Keep size and position of the window when exit',
+  'auto-resume-all': 'Automatically resume all unfinished tasks',
+  'default-dir': 'Default Download Directory',
```

Comment:
Sometimes for layout considerations, you need to simplify the wording. `Default Download Directory` is too long.

Reply:
I updated it to `Default Path`